### PR TITLE
Dijkstra algorithm in go#2652

### PR DESCRIPTION
**Pull request by Basilleaf24**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please complete the PR template

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#Enter the number below which you need to find prime numbers
```
Comment (asha15) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.rb:1]: remove this file

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Don't create multiple PR for the same issue.

**Pull request closed without merging**